### PR TITLE
Add unbalanced polymer databinding delimitors rule.

### PR DESCRIPTION
**Pull request by rictic**
- [x] CHANGELOG.md has been updated

Steals from @usergenic's PR #1

---
This change is [Reviewable](https://reviewable.io/reviews/polymer/polymer-linter/23)

### CODE REVIEW

**Comment by rictic** [comment]
Fixes #30

File: src/polymer/matchers.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+export const isDomBindTemplate =
+    p.AND(p.hasTagName('template'), p.hasAttrValue('is', 'dom-bind'));
+export const isDomModuleTemplate = p.AND(
+    p.parentMatches(p.hasTagName('dom-module')), p.hasTagName('template'));
+export const isTemplate = p.hasTagName('template');
+export const isTemplateDescendant = p.parentMatches(isTemplate);
+
+export interface Template extends parse5.ASTNode { content: parse5.ASTNode; }
+
+export function getNestedDataBindingTemplates(node: parse5.ASTNode) {
```
Comment (usergenic) [inline src/polymer/matchers.ts]: I feel like this needs commenting.  Also, I feel like matchers.ts should be all predicate functions, but maybe that wouldn't make sense for this one, so not a deal-breaker.

**Review by usergenic — CHANGES_REQUESTED** [review]

File: src/polymer/matchers.ts (hunk 1 above)
Comment (rictic) [reply src/polymer/matchers.ts]: It turns out that this can be written more correctly as a predicate (as top level dom-if and dom-repeat is allowed).

Comment (usergenic) [reply src/polymer/matchers.ts]: So clean!  Awesome.

**Review by usergenic — APPROVED** [review]
LGTM

**Pull request merged**